Readme.md

```diff
@@ -51,6 +51,16 @@ _How application logic is executed._
 </summary>
 </details>
 
+<details><summary><b>Networking & Protocols</b></br>
```

Comment:
I think that's worth a dedicated "main" section - I wouldn't say it's a core concept or principle in itself. Maybe `1.7`? We can of course also first merge and then reorder :+1:

Reply:
Sure, we can do that. I also thought that there might be some embedded devices that are not connected to standard IP networking, but it is very common and therefore one of the essentials skills. Better now? 😉